test/xpu_api/numerics/complex.number/complex.transcendentals/asinh.pass.cpp

```diff
@@ -126,7 +126,12 @@ void test_edges()
 
 ONEDPL_TEST_NUM_MAIN
 {
+#if !_PSTL_TEST_COMPLEX_OP_ASINH_USING_DOUBLE
     test<float>();
+#else
+    IF_DOUBLE_SUPPORT(test<float>())
+#endif
```

Comment:
On windows we are not testing anything here, I think the appropriate thing to do would be to report "skipped" rather than "passed" for these tests.

Reply:
I think it's not very simple exercise now. Required to change a lot of functions.
Also we always test code on host.

Reply:
I guess I just don't know what utility this xpu_api test suite provides if we return success from this test when there is no version which works with the environment in question.

Perhaps we need to re-evaluate how we treat these tests within oneDPL. They probably need to be more separated from our oneDPL functional testing, and it is OK if they fail as long as they are an accurate depiction of the "status" of the standard library call within a kernel with a certain environment / hardware.

I wont insist here on changes, but we may need to address this at a higher level.

Reply:
May be you are right, but not in this PR.